source/administration/statistics.rst

```diff
@@ -0,0 +1,66 @@
+Statistics
+================
+
+Statistics on users, posts and channels are tracked for each system and team. Enterprise Editions have access to advanced system statistics.
+
+Site statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Reporting** > **Site Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of accounts created on your system, including both active and inactive accounts.
+
+**Total Teams:** The total number of teams created on your system, including deleted teams.
+
+**Total Posts:** The total number of posts made in all the teams on your system, including deleted posts and posts made using automation.
+
+**Total Channels:** The total number of channels created in all the teams on your system, including deleted channels.
+
+**Total Posts (graph):** The total number of posts made on a certain day in all the teams on your system, including deleted posts and posts made using automation.
+
+**Active Users With Posts (graph):** Users who logged in and made a post on a certain day in all the teams on your system, including system messages posted from the user's account.
```

Comment:
I wonder if it's a bit confusing saying "...logged in and made a post on a certain day...", since really it's just made a post. They didn't specifically log in on that day, and by saying that it seems like this is an important part of the criteria to appear on that graph.